src/services/PushRule.js

```diff
@@ -0,0 +1,40 @@
+import { BaseService, RequestHelper } from '../infrastructure';
+
+class PushRule extends BaseService {
+
+  create(projectId, options) {
+    const pId = encodeURIComponent(projectId);
+
+    return RequestHelper.post(this, `projects/${pId}/push_rule`, options);
+  }
+
+  async createOrEdit(projectId, options) {
```

Comment:
I can remove this if you consider this too much but I thought it be useful otherwise so people did not have to implement this all over.